contrib/verify-commits/verify-commits.sh

```diff
@@ -38,8 +36,8 @@ PREV_COMMIT=""
 
 while true; do
 	if [ "$CURRENT_COMMIT" = $VERIFIED_ROOT ]; then
-		echo "There is a valid path from "$CURRENT_COMMIT" to $VERIFIED_ROOT where all commits are signed!"
-		exit 0;
+		echo "There is a valid path from \"$CURRENT_COMMIT\" to $VERIFIED_ROOT where all commits are signed!"
```

Comment:
According to the line above this is only printed when both vars are the same. No need to print them both.

Reply:
@MarcoFalke Do you have any suggestion regarding the correct output to print? :-)

Reply:
maybe

```diff
 PREV_COMMIT=""
+INITIAL_COMMIT=${CURRENT_COMMIT}
 
 while true; do
 	if [ "$CURRENT_COMMIT" = $VERIFIED_ROOT ]; then
-		echo "There is a valid path from "$CURRENT_COMMIT" to $VERIFIED_ROOT where all commits are signed!"
+		echo "There is a valid path from \"$INITIAL_COMMIT\" to $VERIFIED_ROOT where all commits are signed!"